index.js

```diff
@@ -2485,17 +2550,14 @@
                def);
   }
 
-  var create =
-  def('create',
-      {},
-      [RecordType({checkTypes: Boolean_, env: Array_(Any)}), AnyFunction],
-      _create);
+  var Options = RecordType(Object_, {checkTypes: Boolean_, env: Array_(Any)});
```

Comment:
Oh - I thought we changed that a while ago! When we removed `applyTypes` or something.